assets/css/calypsoify.css

```diff
@@ -1350,4 +1350,44 @@ p.search-box {
 .subsubsub .search-box.is-expanded .search-box__close-icon,
 .subsubsub .search-box.is-expanded input[type="search"] {
 	display: flex;
+}
+
+/* Fix display of the "Please copy and paste this information in your ticket when contacting support" notice displayed on the system status page */
+.woocommerce_page_wc-status .woocommerce-message a.debug-report {
+	color: #0073aa !important;
```

Comment:
Do we want to use buttons here and append to [button list](https://github.com/Automattic/wc-calypso-bridge/blob/36992d60c89c30085b0345b3dc45e9cd9832fe49/assets/css/calypsoify.css#L143) instead like the original design or switch to links?

I think this is your call unless @josemarques feels strongly about it.

Reply:
Original style for reference:
<img width="428" alt="screen shot 2018-11-15 at 5 17 53 am" src="https://user-images.githubusercontent.com/10561050/48513506-eafd3000-e896-11e8-9a93-71175901b11d.png">